apps/internal/local/server.go

```diff
@@ -84,6 +84,14 @@ func New(reqState string, port int) (*Server, error) {
 		return nil, err
 	}
 
+	if len(successPage) > 0 {
+		okPage = successPage
+	}
+
+	if len(errorPage) > 0 {
+		failPage = errorPage
+	}
```

Comment:
`WithSystemBrowserOptions` is a per-call option but `failPage` and `okPage` are package-level variables. Setting them here therefore affects every `AcquireTokenInteractive` call, for all clients.

Reply:
@chlowell I fixed this one, by moving the scope to the server struct;